lib/services/api_service.dart

```diff
@@ -68,6 +68,89 @@ class ApiService {
     return _user!;
```

Comment:
Login with Password still returns the password which will be saved too.
What I didn't check is, if the user gets redirected to the login after the token has expired. 
And the token doesn't get to be refreshed yet.

Reply:
I reverted the change that just reconnects with the token to call the login if there is a password.

If the token has expired (not sure when/how that happens as it appears to be an opaque blob and not a jwt), it bubbles up as a 401 that will cause the app to error out and send you to the login